airflow/providers/http/operators/http.py

```diff
@@ -49,6 +50,12 @@ class SimpleHttpOperator(BaseOperator):
     :param data: The data to pass. POST-data in POST/PUT and params
         in the URL for a GET request. (templated)
     :param headers: The HTTP headers to be added to the GET request
+    :param pagination_function: A callable that generate the parameters used
+        to call the API again. Typically used when the API is paginated and
+        returns a cursor, a next page id, or a next page URL. When provided,
+        the Operator will call the API repeatedly until this function returns
+        None. Also, by default, the result of the Operator will be a list of
+        Response.text objects.
     :param response_check: A check against the 'requests' response object.
```

Comment:
Just a minor thing :)
The docstring for `response_check` and `response_filter` should reflect the fact that the response could be a list of responses when the `pagination_function` is passed.

Reply:
Updated it, thanks !